Packs/iDefense/Integrations/FeediDefense/FeediDefense.py

```diff
@@ -0,0 +1,115 @@
+from typing import Dict, Union
+from CommonServerPython import *
+from JSONFeedApiModule import *  # noqa: E402
+
+
+def custom_build_iterator(client: Client, feed: Dict, limit, **kwargs) -> List:
+    if not limit:
+        limit = 50000
+    params: dict = feed.get('filters', {})
+    current_indicator_type = feed.get('indicator_type')
+    integration_context = get_integration_context()
+    page_number = integration_context.get(f'{current_indicator_type}_page', 1)
+
+    more_indicators = True
+    is_limit_reached = False
+    params['page_size'] = 200
+    result: list = []
+    while more_indicators and not is_limit_reached:
+        params['page'] = page_number
+        r = requests.get(
+            url=feed.get('url', client.url),
+            verify=client.verify,
+            auth=client.auth,
+            cert=client.cert,
+            headers=client.headers,
+            params=params,
+            **kwargs
+        )
+
+        try:
+            r.raise_for_status()
+            data = r.json()
+            result = result + jmespath.search(expression=feed.get('extractor'), data=data)
+            more_indicators = data.get('more')
+            page_number += 1
+            if not more_indicators:
+                set_integration_context({f'{current_indicator_type}_page': 1})
+            if len(result) >= limit:
+                is_limit_reached = True
+                set_integration_context({f'{current_indicator_type}_page': page_number})
```

Comment:
I would just break from the loop, i think it is more readable.
```suggestion
            if len(result) >= limit:
                set_integration_context({f'{current_indicator_type}_page': page_number})
                break
```

Reply:
also, please add inline docstring for the dev that will follow